### PR TITLE
Anilist importer

### PR DESCRIPTION
**Pull request by toyhammered**
Still need to implement notes. Having issues with my `#each` test not actually sure _why_ it is returning an error (at the `yield row.media`)
@NuckChorris

### CODE REVIEW

File: spec/support/factory_girl.rb [hunk 1]
```diff
@@ -1,14 +0,0 @@
-require 'factory_girl'
```
Comment (toyhammered) [inline spec/support/factory_girl.rb:1]: hmmm this is most likely a screw up on my end, accidentally added my version and removed it using git rm before commiting.

File: app/models/list_import/anilist.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    private
+
+    def get_auth_token
+      url = "#{ANILIST_API}auth/access_token"
+      request = Typhoeus::Request.new(url,
+        method: :post,
+        body: {
+          grant_type: 'client_credentials',
+          client_id: 'toyhammered-c6imc',
```
Comment (toyhammered) [inline app/models/list_import/anilist.rb:70]: should these be added to `.env`? (I don't really care it is only `GET` access anyways)

Comment (NuckChorris) [reply app/models/list_import/anilist.rb:70]: Would be good to read from `ENV['ANILIST_CLIENT_ID']` and `ENV['ANILIST_CLIENT_SECRET']`, but this works for now

**Review by NuckChorris — APPROVED** [review]

**Pull request merged**